python/MoltresXS.py

```diff
@@ -0,0 +1,280 @@
+"""
+     Python Function that takes homoginezed cross sections and prepares them for MOLTRES
+
+"""
+
+import io
+import sys
+import numpy as np
+import h5py
```

Comment:
PyNE uses pytables (`import tables`) on the backend. 
So, to have pyne, the user certainly already has pytables.
To reduce the number of dependencies, I'd recommend using pytables rather than h5py.